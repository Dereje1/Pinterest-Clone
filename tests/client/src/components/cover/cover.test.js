import React from 'react';
import { shallow } from 'enzyme';
import toJson from "enzyme-to-json";
import Cover from '../../../../../client/src/components/cover/cover';

describe('The landing page', () => {
    test('will display the cover', () => {
        const wrapper = shallow(<Cover />);
        expect(toJson(wrapper)).toMatchSnapshot();
    })
})
