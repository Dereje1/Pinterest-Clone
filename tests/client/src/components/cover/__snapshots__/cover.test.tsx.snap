// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The landing page will display the cover 1`] = `
<div
  id="cover"
>
  <div
    id="logincard"
  >
    <div>
      <div
        id="logo"
      >
        <i
          aria-hidden="true"
          className="fa fa-pinterest"
        />
      </div>
      <div
        id="welcome"
      >
        Welcome to Pinterest
      </div>
      <div
        id="subheader"
      >
        Find new ideas to try
      </div>
      <div
        id="disclaimer"
      >
        Cloned For Educational Purposes Only!
      </div>
      <div
        id="gitsource"
      >
        <a
          href="https://github.com/Dereje1/Pinterest-Clone"
          rel="noopener noreferrer"
          target="_blank"
        >
          <i
            aria-hidden="true"
            className="fa fa-github"
          />
           Github
        </a>
      </div>
    </div>
    <div>
      <Connect(LoginButtons)
        guest={[Function]}
      />
    </div>
  </div>
</div>
`;
